### ISSUE TITLE
When running tests on klarGeckoDebug mode, the app crashes with strictmode violation

### ISSUE THREAD

**Issue by npark-mozilla** [mozilla-mobile/focus-android#2182]
Upon running the test, it shows "The app has violated its self-enforced StrictMode Policy" dialog.

@pocmo a couple questions:
- Should we run tests on focusGeckoDebug, or klarGeckoDebug?  I've been trying to run tests on klar and rename the dependencies to Klar for the tests in AndroidTestGecko folder.
- Wasn't there something we did to avoid running into strict mode violation in webview? we'd need a same fix here.

**Comment by pocmo** [comment]
> Should we run tests on focusGeckoDebug, or klarGeckoDebug? I've been trying to run tests on klar and rename the dependencies to Klar for the tests in AndroidTestGecko folder.

Technically it shouldn't matter for now. However as we are running tests on focusWebView - let's try klarGecko just to make sure there are no surprises in "klar".

> Wasn't there something we did to avoid running into strict mode violation in webview? we'd need a same fix here.

I saw this dialog a bunch of times too. There might actually be a problem here that strict mode found and we need to fix.

I think we didn't disable strict mode completely. We just changed it from "crash" to "show dialog". This here might be a problem that only shows up in the GeckoView builds.
